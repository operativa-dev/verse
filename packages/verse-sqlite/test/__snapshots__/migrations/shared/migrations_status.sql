-- Executing SQL: Parameters: [$1='__verse_migrations']
select 1
from "sqlite_master"
where ("type" = 'table') and ("name" = ?)

