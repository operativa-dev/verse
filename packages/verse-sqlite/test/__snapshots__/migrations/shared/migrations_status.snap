{
  "database": "/Users/andrewpeters/Development/Operativa/verse/packages/verse-sqlite/test/migrations_status.sqlite",
  "databaseExists": true,
  "directory": "/Users/andrewpeters/Development/Operativa/verse/packages/verse-core/dist/test/migrations/migrations",
  "directoryExists": true,
  "migrations": [
    {
      "applied": false,
      "file": "2023-11-15-08-41-Initial.js",
      "id": "2023-11-15-08-41-Initial",
      "name": "Initial",
    },
    {
      "applied": false,
      "file": "2023-11-16-08-42-Second.js",
      "id": "2023-11-16-08-42-Second",
      "name": "Second",
    },
    {
      "applied": false,
      "file": "2023-11-18-08-42-AllOperations.js",
      "id": "2023-11-18-08-42-AllOperations",
      "name": "AllOperations",
    },
  ],
  "tableExists": false,
}